daemon/execdriver/windows/run.go

```diff
@@ -75,8 +75,13 @@ func (d *Driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	}
 
 	for i := 0; i < len(c.LayerPaths); i++ {
+		_, filename := filepath.Split(c.LayerPaths[i])
```

Comment:
Just this part of the changes to this file (generating the layer ID using hcsshim.NameToGuid) are part of the graphdriver refactor.  This is a bug fix to make sure docker is passing the same layer IDs as what gets generated by other Windows components.